src/ShopCurrencyBundle/Bridge/Chameleon/Migration/Script/update-1545232400.inc.php

```diff
@@ -0,0 +1,155 @@
+<h1>Build #1545232400</h1>
+<h2>Date: 2018-12-19</h2>
+<div class="changelog">
+    - -#234 add currency field to used voucher
+</div>
+<?php
+
+$fieldId = TCMSLogChange::createUnusedRecordId('cms_field_conf');
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+  ->setFields([
+      'cms_tbl_conf_id' => TCMSLogChange::GetTableId('shop_voucher_use'),
+      'name' => 'new_field',
+      'translation' => '',
+      'cms_field_type_id' => TCMSLogChange::GetFieldType('CMSFIELD_STRING'),
+      'cms_tbl_field_tab' => '',
+      'isrequired' => '0',
+      'fieldclass' => '',
+      'fieldclass_subtype' => '',
+      'class_type' => 'Core',
+      'modifier' => 'none',
+      'field_default_value' => '',
+      'length_set' => '',
+      'fieldtype_config' => '',
+      'restrict_to_groups' => '0',
+      'field_width' => '',
+      'position' => '0',
+      '049_helptext' => '',
+      'row_hexcolor' => '',
+      'is_translatable' => '0',
+      'validation_regex' => '',
+      'id' => $fieldId,
+
+  ])
+;
+TCMSLogChange::insert(__LINE__, $data);
+
+$query ="ALTER TABLE `shop_voucher_use`
+                        ADD `value_used_in_order_currency` DECIMAL(10,2) NOT NULL COMMENT 'Value consumed in the order currency: Displays the consumption value of the voucher in the currency in which the order was made at the time of ordering.'";
+TCMSLogChange::RunQuery(__LINE__, $query);
+
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
```

Comment:
Please cleanup updates, so that insert statements directly create complete records.

Reply:
changed